docs/OV_Runtime_UG/deployment/local-distribution.md

```diff
@@ -9,29 +9,29 @@ Local dsitribution is also appropriate for OpenVINO binaries built from sources
 ### C++ or C language
```

Comment:
```suggestion
> **NOTE**: The steps below are operating system independent and refer to a library file name without any prefixes (like `lib` on Unix systems) or suffixes (like `.dll` on Windows OS). Do not put `.lib` files on Windows OS to the distribution, because such files are needed only on a linker stage.

### C++ or C language
```